src/pages/article-page/mocks/index.ts

```diff
@@ -0,0 +1,56 @@
+import type { IArticlePageData } from "../models";
+import CoverImage from '@/shared/assets/images/resources/articleCover.png'
+export const mockArticlePageData: IArticlePageData = {
+    author: "Иванов Иван",
+    date: new Date('2025-10-11').toISOString(),
+    title: "Как научиться говорить о своих чувствах честно?",
+    content: [
+        {
+            type: "p",
+            data: "Честно ли вы говорите о своих чувствах? Или чаще всего" +
+            " используете обвинения, упрёки и сарказм, чтобы донести, " + 
+            "что вас что-то задело? Это не упрёк, просто вопрос."
+        },
+        {
+            type: "image",
+            src: CoverImage,
+            alt: "Cover image"
+        },
+        {
+            type: "p",
+            data: 'В современном мире, где коммуникация становится' +
+    'все более важной, умение открыто и честно говорить о своих' + 
+    'чувствах — это навык, который может значительно улучшить ' + 
+    'качество Ваших отношений и повысить эмоциональное благополучие.' + 
+    'Однако многие люди сталкиваются с трудностями при выражении ' +
+    'своих эмоций. В этой статье мы рассмотрим несколько стратегий, ' +
+    'которые помогут Вам научиться говорить о своих чувствах честно и открыто.'
+        },
+        {
+            type: "h2",
+            data: "Понимание своих чувств"
+        },
+        {
+            type: "block",
+            data: [
+                {
+                    type: "p",
+                    data: "Прежде чем начать говорить о своих чувствах," +
+                " важно научиться их распознавать и понимать. Задайте себе вопросы:"
+                },
+                {
+                    type: "ul",
+                    items: [
+                        'Что я чувствую в данный момент?', 
+                        'Почему я так себя чувствую?', 
+                        'Какое событие или ситуация вызвали эти чувства?'
+                    ]
+                },
+                {
+                    type: "p",
+                    data: "Эти вопросы помогут вам лучше осознать свои эмоции и подготовиться к их выражению."
```

Comment:
```suggestion
                    data: "Ведение дневника эмоций может быть полезным инструментом для анализа своих чувств. Записывая свои мысли и эмоции, Вы сможете лучше понять их природу и причины."
```

Чтобы оно уж совсем соответствовало макету для проверок